docs/release-notes.md

```diff
@@ -44,8 +44,9 @@ nav_order: 9
 - Drop `extensions` section _(fcos 1.5.0+, openshift 4.13.0+)_
 - Drop `LuksOption` and `RaidOption` types _(Go API for fcos 1.5.0+,
   flatcar 1.1.0+, openshift 4.14.0-experimental)_
-- Require Go 1.18+
-
+- Require Go 1.18+ 
+- Add warning for bad non-root partition tables when 'wipe_tables' is false and partition number is zero in _(fcos 1.6.0-exp)_
```

Comment:
hmm, how about 

"Warn if specifying a partition number of zero on a non root device with `wipe_tables` set to false \_(fcos 1.6.0-exp)\_"

Reply:
Actually, I think I changed my mind.
WDYT
"Warn on invalid partition configuration \_(fcos 1.6.0-exp)\_"

Reply:
"Warn on invalid partition configuration _(fcos 1.6.0-exp)_" sounds good :+1: